client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -6,26 +6,31 @@ const _ = require('lodash');
 const request = require('superagent');
 const { Meta } = require('vitreum/headtags');
 
+const Markdown = require('naturalcrit/markdown.js');
+
 const Nav = require('naturalcrit/nav/nav.jsx');
 const Navbar = require('../../navbar/navbar.jsx');
 
-const NewBrew = require('../../navbar/newbrew.navitem.jsx');
-const ReportIssue = require('../../navbar/issue.navitem.jsx');
-const PrintLink = require('../../navbar/print.navitem.jsx');
-const Account = require('../../navbar/account.navitem.jsx');
+const NewBrewNavItem = require('../../navbar/newbrew.navitem.jsx');
+const PrintLinkNavItem = require('../../navbar/print.navitem.jsx');
+const ReportIssueNavItem = require('../../navbar/issue.navitem.jsx');
 const RecentNavItem = require('../../navbar/recent.navitem.jsx').both;
+const AccountNavItem = require('../../navbar/account.navitem.jsx');
 
 const SplitPane = require('naturalcrit/splitPane/splitPane.jsx');
 const Editor = require('../../editor/editor.jsx');
 const BrewRenderer = require('../../brewRenderer/brewRenderer.jsx');
 
-const Markdown = require('naturalcrit/markdown.js');
-
 const googleDriveActive = require('../../googleDrive.png');
 const googleDriveInactive = require('../../googleDriveMono.png');
 
 const SAVE_TIMEOUT = 3000;
 
+const BREWKEY = 'homebrewery-new';
+const STYLEKEY = 'homebrewery-new-style';
+
+const editTypes = ['edit', 'new'];
```

Comment:
I don't think this is necessary. We shouldn't ever have a case when an invalid editType comes through since the only place it can come from is us, at `homebrewery.jsx`